app/model/db/notification.py

```diff
@@ -84,6 +84,10 @@ class Notification(Base):
     deleted_at = Column(DateTime, default=None)
 
     # 通知が発生した日付（blockTime）
+    # NOTE:
+    #  Postgres: Stored as UTC datetime.
+    #  MySQL: Before 23.3, stored as JST datetime.
+    #         From 23.3, stored as UTC datetime.
     block_timestamp = Column(DateTime)
```

Comment:
- Added comment about the way in which block_timestamp is stored.
  - With **Postgres**, block_timestamp was stored as UTC timestamp.
  - With **MySQL**, block_timestamp was stored as JST timestamp.
    - In this PR, fixed batches to store block_timestamp as UTC timestamp.